pypesto/engine/base.py

```diff
@@ -13,13 +13,16 @@ def __init__(self):
         pass
 
     @abc.abstractmethod
-    def execute(self, tasks: List[Task]):
+    def execute(self, tasks: List[Task], progress_bar: bool = True):
         """Execute tasks.
 
         Parameters
         ----------
         tasks:
             List of tasks to execute.
+        progress_bar:
+            Indicates, whether a progress bar should be displayed.
+            Default is True.
```

Comment:
```suggestion
            If `True`, a progress bar is displayed.
```
No need for default since it's already in the signature

Reply:
Same edit can be made elsewhere

Reply:
sentence can be omitted entirely too ;)

Reply:
for maximum brevity :sunglasses: